manifest.json

```diff
@@ -1,4501 +1,4572 @@
-											],
-											"supports": [
-												"regionByPct",
-												"sizeByForcedWh",
-												"sizeByWh",
-												"sizeAboveFull",
-												"rotationBy90s",
-												"mirroring",
-												"gray"
-											]
-										}
-									]
-								},
-								"height": 7350,
-								"width": 5250
-							},
-							"on": "http://fauvel.archivengine.com/canvas/23r"
-						}
-					]
-				},
-				{
-					"@id": "http://fauvel.archivengine.com/canvas/23v",
-					"@type": "sc:Canvas",
-					"label": "Folio 23v",
-					"height": 7350,
-					"width": 5250,
-					"images": [
-						{
-							"@context": "http://iiif.io/api/presentation/2/context.json",
-							"@id": "http://fauvel.archivengine.com/annotation/23v-image",
-							"resource": {
-								"@id": "https://iip.textlab.org/?IIIF=fauvel/58.tiff/full/full/0/default.jpg",
-								"@type": "dctypes:Image",
-								"format": "image/jpeg",
-								"service": {
-									"@context": "http://iiif.io/api/image/2/context.json",
-									"@id": "https://iip.textlab.org/?IIIF=fauvel/58.tiff",
-									"profile": [
-										"http://iiif.io/api/image/2/level1.json",
-										{
-											"formats": [
-												"jpg"
-											],
-											"qualities": [
-												"native",
-												"color",
-												"gray"
-											],
-											"supports": [
-												"regionByPct",
-												"sizeByForcedWh",
-												"sizeByWh",
-												"sizeAboveFull",
-												"rotationBy90s",
-												"mirroring",
-												"gray"
-											]
-										}
-									]
-								},
-								"height": 7350,
-								"width": 5250
-							},
-							"on": "http://fauvel.archivengine.com/canvas/23v"
-						}
-					]
-				},
-				{
-					"@id": "http://fauvel.archivengine.com/canvas/24r",
-					"@type": "sc:Canvas",
-					"label": "Folio 24r",
-					"height": 7350,
-					"width": 5250,
-					"images": [
-						{
-							"@context": "http://iiif.io/api/presentation/2/context.json",
-							"@id": "http://fauvel.archivengine.com/annotation/24r-image",
-							"resource": {
-								"@id": "https://iip.textlab.org/?IIIF=fauvel/59.tiff/full/full/0/default.jpg",
-								"@type": "dctypes:Image",
-								"format": "image/jpeg",
-								"service": {
-									"@context": "http://iiif.io/api/image/2/context.json",
-									"@id": "https://iip.textlab.org/?IIIF=fauvel/59.tiff",
-									"profile": [
-										"http://iiif.io/api/image/2/level1.json",
-										{
-											"formats": [
-												"jpg"
-											],
-											"qualities": [
-												"native",
-												"color",
-												"gray"
-											],
-											"supports": [
-												"regionByPct",
-												"sizeByForcedWh",
-												"sizeByWh",
-												"sizeAboveFull",
-												"rotationBy90s",
-												"mirroring",
-												"gray"
-											]
-										}
-									]
-								},
-								"height": 7350,
-								"width": 5250
-							},
-
```

Comment:
This one entry missed the update to using the new data structure for annotation lists.

Reply:
Hey @jamiefolsom, looks great! Just noted one spot in manifest where our previous test entry (1v) dodged the update script. Otherwise all good.